media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
```

Comment:
@alexgibson It seems that we should be good on this in general, as far back as IE8:
http://caniuse.com/#search=border-box